exporter/cassandraexporter/factory.go

```diff
@@ -26,6 +26,7 @@ func createDefaultConfig() component.Config {
 	return &Config{
 		DSN:        "127.0.0.1",
 		Port:       9042,
+		Timeout:    10,
```

Comment:
Is there a reason why this doesn't match the default specified in the README and examples provided? The actual default will be what's defined here.